src/App.js

```diff
@@ -1,5 +1,87 @@
+import { MissionUtils } from "@woowacourse/mission-utils";
+
 class App {
-  async play() {}
+
+  nameValidation(input) {
```

Comment:
```suggestion
  validateCarNames(input) {
```
일부 케이스를 제외하고는 보통 메소드나 함수명은 동사형으로 짓는 것 같은데, 동사형 이름은 어떠신가요?